src/NuGet.Core/NuGet.Build.Tasks.Pack/GetPackOutputItemsTask.cs

```diff
@@ -33,6 +33,8 @@ public class GetPackOutputItemsTask : Microsoft.Build.Utilities.Task
 
         public string SymbolPackageFormat { get; set; }
 
+        public bool OutputFileNamesWithoutVersion  { get; set; }
```

Comment:
```suggestion
        public bool OutputFileNamesWithoutVersion { get; set; }
```
CI is failing because of this extra whitespace...

Reply:
Fixed. I guess the analyzers don't run on the local debug build?